x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/risk_score_entity_calculation.ts

```diff
@@ -76,7 +76,8 @@ export default ({ getService }: FtrProviderContext): void => {
     });
   };
 
-  describe('@ess @serverless @serverlessQA Risk Scoring Entity Calculation API', () => {
+  describe('@ess @serverless @serverlessQA Risk Scoring Entity Calculation API', function () {
```

Comment:
Just curious, is there a reason this can't be implemented by utilizing more annotations, something like `@esGate`?

Reply:
@jaredburgettelastic it is not a mocha tagging. It is a custom tagging implementation introduced by FTR.